src/models/users/useGetMe.js

```diff
@@ -1,39 +1,23 @@
-import axios from 'axios';
 import { get } from 'lodash-es';
-import { useQuery } from 'react-query';
 import * as Sentry from '@sentry/react';
 import queryKeys from '../../constants/queryKeys';
+import useGet from '../../hooks/useGet';
 
 export default function useGetMe() {
-  const result = useQuery(
-    queryKeys.me,
-    async () => {
-      const response = await axios.request({
-        url: `${__houston_url__}/api/v1/users/me`,
-        method: 'get',
+  function onSuccess(response) {
+    if (!__DEV__) {
```

Comment:
?

Reply:
I don't understand what __DEV__ is or where it's coming from. I also get the eery feeling that I have asked this exact question before.

Reply:
this is a build time variable that is set up in webpack. if you're running the app in its dev environment `__DEV__ === true`, if you are running the production build `__DEV__ ===  false`

Reply:
Thanks!